velox/functions/prestosql/StringFunctions.cpp

```diff
@@ -143,7 +143,7 @@ class UpperLowerTemplateFunction : public exec::VectorFunction {
  * */
 class ConcatFunction : public exec::VectorFunction {
  public:
-  bool isDefaultNullBehavior() const override {
+  bool propagateStringEncodingFromAllInputs() const override {
```

Comment:
will it still retain default null behavior?

Reply:
Yes, I think by default `isDefaultNullBehavior()` returns true. (See exec::VectorFunction.)

Reply:
Yes. The default null behavior is the default :-)